### ISSUE TITLE
Locked in FailSafe mode since upgrade to 20220606~ynh1

### ISSUE THREAD

**Issue by glonguetaud** [YunoHost-Apps/ttrss_ynh#131]
### Describe the bug

TTRSS is locked in FailSafe mode since upgrade to 20220606~ynh1

### Context

- Hardware: *Server at home*
- YunoHost version: 4.3.6.3
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*
- If upgrading, current package version: *20220606~ynh1*

### Steps to reproduce

- *After updating to 20220606~ynh1 via Webmin*

1.  Go to 'https://my-domain/ttrss'
2.  ttrss opens in FailSafe Mode : 

      > Tiny Tiny RSS est en mode sans-échec. Tous les thèmes et les greffons sont désactivés. Vous devrez vous déconnecter et vous reconnecter pour désactiver ce mode. 
3. I cannot disconnect...
4. Looking at ```sudo journalctl -u ttrss --since "2022-06-09 10:00"``` shows that all actions are performed by php7.3 before update, and by php8.0 after.

### Expected behavior

I installed custom themes and activated plugins that don't work anymore.

Thank you !!

**Comment by glonguetaud** [comment]
...More info from the admin page/System : 

<html>
<body>


E_COMPILE_ERROR (64) | plugins/af_proxy_http/init.php:36 | Declaration of Af_Proxy_Http::hook_enclosure_entry($enc) must be compatible with Plugin::hook_enclosure_entry($entry, $id, $rv)  Remote IP: 192.168.0.254 Request URI: /ttrss/ User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0
-- | -- | --



</body>
</html>

**Comment by glonguetaud** [comment]
I disabled it manually (moved it from the console in the plugins folder) as I couldn't do this with the admin interface (`Error while loading plugins list: RequestError: Unable to load backend.php status: 500.`) 

For the record, I had to give me (as user) the administrator permission, through `Users → Acces Level`  to be able to disconnect as a regular user (as requested while in failsafe mode). 
That ability remains after returning to normal user. 

Everything is restored (except the af_proxy_http plugin) now. 
Thank for the help !
